tests/test-kernels/src/bin/gdb.rs

```diff
@@ -4,6 +4,8 @@ use hermit as _;
 static mut WATCH: u8 = 2;
 
 fn main() {
+	println!("Hello from gdb!");
```

Comment:
This breaks the test somehow, possibly because of some offset changing - I will investigate after I get a positive reaction to this change.